src/Components/TodoList/ListItem/ListItem.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import { Box, Button, Checkbox, Typography } from "@mui/material";
+import React from "react";
+import { COLORS } from "../../../colors";
 
-export default function ListItem() {
+const removeButtonSx = {
+  bgcolor: COLORS.RED_BUTTON,
+  textTransform: "none",
+  "&:hover": {
+    backgroundColor: COLORS.RED_BUTTON_HOVER,
+  },
+};
+
+interface ListItemProps {
+  description: string;
```

Comment:
To make this component generic, please add removeButtonExtraSx prop, its type should be SxProps.
Use this prop to append additional styles to the button.